python/ray/dashboard/consts.py

```diff
@@ -26,7 +26,7 @@
     "RAY_DASHBOARD_STATS_PURGING_INTERVAL", 60 * 10
 )
 RAY_DASHBOARD_STATS_UPDATING_INTERVAL = env_integer(
-    "RAY_DASHBOARD_STATS_UPDATING_INTERVAL", 2
+    "RAY_DASHBOARD_STATS_UPDATING_INTERVAL", 15
```

Comment:
@alanwguo if we want to preserve UI update frequency to 2s we'll need to switch to pulling model in the UI

Reply:
I think 15s is probably fine.